### PR TITLE
Use setup_instance script to enable conda and load cuda libraries

### PR DESCRIPTION
**Pull request by xuzhao9**
Fixes the broken torchbench CI after the machine image update.
RUN_TORCHBENCH: nvfuser

### CODE REVIEW

File: .github/workflows/run_torchbench.yml [hunk 1]
```diff
@@ -36,17 +37,18 @@ jobs:
...
           # shellcheck disable=SC1091
-          . "${HOME}"/anaconda3/etc/profile.d/conda.sh
+          . "${SETUP_SCRIPT}"
           conda activate pr-ci
           # pin cmake version to 3.22 since 3.23 breaks pytorch build
           # see details at: https://github.com/pytorch/pytorch/issues/74985
           conda install -y numpy="${NUMPY_VERSION}" requests ninja pyyaml mkl mkl-include \
                            setuptools cmake=3.22 cffi typing_extensions boto3 \
                            future six dataclasses pillow pytest tabulate gitpython git-lfs tqdm psutil
+         pip install --pre torch torchvision torchtext -f https://download.pytorch.org/whl/nightly/cu116/torch_nightly.html
```
Comment (davidberard98) [inline .github/workflows/run_torchbench.yml:47]: Needs to be indented 1 space

**Comment by xuzhao9** [comment]
@davidberard98  can I get a stamp on this PR?

**Review by davidberard98 — APPROVED** [review]

**Pull request closed without merging**